app/assets/stylesheets/application.css.scss

```diff
@@ -13,3 +13,40 @@
  *= require_tree .
  *= require_self
  */
+
+ #container {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 1 spaces